src/ParserConfig.cc

```diff
@@ -50,6 +50,10 @@ class sdf::ParserConfig::Implementation
   /// reading the SDF/URDF file.
   public: bool preserveFixedJoint = false;
 
+  /// \brief Flag to convert of links with small (equal to or less than
+  /// minimumAllowedLinkMass) or no mass in URDF to frames in SDF.
+  public: bool convertLinkWithNoMassToFrame = true;
```

Comment:
Now that we're not really creating a new frame in this PR, do we even need this flag?

Reply:
No we won't need it anymore, I've removed it in my current implementation